tests/trafficmanagement/scripts/tcp_trafficshifting_setup.txt

```diff
@@ -32,3 +32,12 @@ $ kubectl label namespace istio-io-tcp-traffic-shifting istio-injection=enabled
 # $snippet deploy_tcp_echo_v1.sh
 $ kubectl apply -f @samples/tcp-echo/tcp-echo-services.yaml@ -n istio-io-tcp-traffic-shifting
 # $endsnippet
+
+# Add the TCP port to the ingress-gateway
+kubectl -n istio-system patch service istio-ingressgateway --patch "
```

Comment:
I haven't looked at the full test, but I'm not sure this is the right place to do this. These sorts of things should happen in the config step of the test, ie set the profile to demo there. But even so, do we really require this port? If that's the case, it must be mentioned on the docs page as well